src/tools-download.ts

```diff
@@ -87,7 +87,9 @@ export async function downloadAndExtract(
 
   const compressionMethod = tar.inferCompressionMethod(codeqlURL);
 
-  if (compressionMethod === "zstd" && process.platform === "linux") {
+  // TODO: Re-enable streaming when we have a more reliable way to respect proxy settings.
+  // eslint-disable-next-line no-constant-condition, no-constant-binary-expression
+  if (false && compressionMethod === "zstd" && process.platform === "linux") {
```

Comment:
Can you turn `false` into an environment variable? This way at least we can selectively enable it.

Maybe something like:

```suggestion
  if (process.env[EnvVar.STREAM_BUNDLE_DOWNLOAD] !== "true" && compressionMethod === "zstd" && process.platform === "linux") {
```

And then add the variable to the `EnvVar` class?